charts/dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_deployment.yaml

```diff
@@ -92,6 +92,16 @@ spec:
       - name: cert
         secret:
           secretName: dapr-sidecar-injector-cert
+      affinity:
```

Comment:
Do we need affinity property here? As @yaron2 mentioned, we can expose global.values to users to enable hybrid mode..

1. add global value to https://github.com/dapr/dapr/blob/master/charts/dapr/values.yaml
2. add nodeSelector conditionally like below
```
{{- if eq .Values.global.hybrid_node true }}
  nodeSelector:
        "beta.kubernetes.io/os": {{ .Values.global.daprControlPlaneOs }}
{{- end }}
```

so user can specify the hybrid mode when he or she deploy it to hybrid cluster
e.g. 
```
helm install dapr dapr/dapr --set global.hybrid_node=true
```

@yaron2 are you agree with this way?

Reply:
I agree, I also think we don't need the `hybrid_node`, we can just do:

`helm install dapr dapr/dapr --set global.daprControlPlaneOs =windows`

```
{{- if eq .Values.global.daprControlPlaneOs true }}
  nodeSelector:
        "beta.kubernetes.io/os": {{ .Values.global.daprControlPlaneOs }}
{{- end }}
```

What do you think?

Reply:
yeah looks simpler to me :)

in this case we do not need `eq` :) 
```
{{- if .Values.global.daprControlPlaneOs }}
...
{{- end }}
```

Reply:
Why would we require manual action by the user, when we can just get the correct behavior automatically?

Reply:
> Why would we require manual action by the user, when we can just get the correct behavior automatically?

I just read the affinity vs nodeSelector rules again, looks like the current implementation has it covered.
@youngbupark in this case I don't see a reason to put the if statement at all, as the affinity is hint in this case.

Reply:
If it works with older versions of k8s (minimum version that we document), I am ok with it.

Reply:
Yeah that should be the case.